tools/web/build-web-assets.sh

```diff
@@ -0,0 +1,17 @@
+# build web assets with anki submodule in rslib-bridge
+
+SRC_DIR=$(pwd)
+ANKI_SRC=$SRC_DIR/rslib-bridge/anki
+BUILD_DIR=$SRC_DIR/rsdroid/build/generated/anki_artifacts
+
+# build for ts
+cd $ANKI_SRC/ts
+bazel build ts --symlink_prefix=/tmp/.bazel/
+
+# build for web
+cd $ANKI_SRC/qt/aqt/data/web/pages
+bazel build pages --symlink_prefix=/tmp/.bazel/
+
+# copy pages to build/generated
+mkdir -p $BUILD_DIR/web
+sudo cp -r /tmp/.bazel/bin/qt/aqt/data/web/pages/* $BUILD_DIR/web
```

Comment:
```suggestion
cp -r /tmp/.bazel/bin/qt/aqt/data/web/pages/* $BUILD_DIR/web
chmod -R a+w $BUILD_DIR/web
```
rm the existing folder to fix the problem, then it should work going forward

Reply:
The `$BUILD_DIR/web` dir created before copying files.